cumulus/tasks/sf-sns-report/schemas/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "SfSnsReportConfig",
+  "description": "Describes the config used by the sf-sns-report task",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
```

Comment:
I assume that all of these properties are required, so they should be marked as required here.

Reply:
Based on the code, they are only needed when the task is the last step of the step function and a granule state is set.